samples/grid-lite/demo/parse-data/demo.js

```diff
@@ -0,0 +1,100 @@
+const initGrid = data => {
+    Grid.grid('container', {
+        dataTable: {
+            columns: data
+        },
+        rendering: {
+            columns: {
+                distribution: 'mixed'
+            }
```

Comment:
```suggestion
            rows: {
                strictHeights: true
            }
```
columns.distribution is not needed, (turned on automatically if `width` exists) and potentially adds confusion. But `strictHeights` is nice, to prevent some ugly line/word breaks.